XCUITests/SyncFAUITests.swift

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+class SyncUITests: BaseTestCase {
+    func testUIFromSettings () {
+        navigator.goto(FxASigninScreen)
+        waitforExistence(app.webViews.staticTexts["Sign in"])
+        XCTAssertTrue(app.navigationBars["Client.FxAContentView"].exists)
+        XCTAssertTrue(app.webViews.textFields["Email"].exists)
+        XCTAssertTrue(app.webViews.secureTextFields["Password"].exists)
+        XCTAssertTrue(app.webViews.buttons["Sign in"].exists)
+    }
+
+    func testPlaceholderValues () {
+        navigator.goto(FxASigninScreen)
+        waitforExistence(app.webViews.staticTexts["Sign in"])
+        let mailPlaceholder = "Email"
+        let passwordPlaceholder = "Password"
+
+        let defaultMailPlaceholder = app.webViews.textFields["Email"].placeholderValue!
+        let defaultPasswordPlaceholder = app.webViews.secureTextFields["Password"].placeholderValue!
+        XCTAssertTrue(mailPlaceholder == defaultMailPlaceholder, "The mail placeholder does not show the correct value")
+        XCTAssertTrue(passwordPlaceholder == defaultPasswordPlaceholder, "The password placeholder does not show the correct value")
+    }
+
+    func testTypeOnGivenFields() {
+        navigator.goto(FxASigninScreen)
+        waitforExistence(app.webViews.staticTexts["Sign in"])
+
+        // Tap Sign in without any value in email Password focus on Email
+        navigator.performAction(Action.TapOnSignInButton)
+        waitforExistence(app.webViews.staticTexts["Valid email required"])
+
+        // Enter only email, wrong and correct and tap sign in
+        navigator.performAction(Action.TypeInvalidEmail)
```

Comment:
I was thinking, perhaps it's better to specify valid/invalid input texts at the script level, rather than having it defined in the Fxscreengraph? I think that would make the tests more readable, and easy to modify in the future.

Reply:
👍 

My suggestion above would be a good place to start.